cvmfs/statistics_database.cc

```diff
@@ -0,0 +1,225 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "statistics_database.h"
+
+
+struct Stats {
```

Comment:
If possible, wrap into anonymous namespace or make part of the `StatisticsDatabase` class

Reply:
I agree, it should go into anonymous namespace, since it's a utility struct only used inside `statistics_database.cc`.